tests/spec/fs.mkdir.spec.js

```diff
@@ -45,4 +45,16 @@ describe('fs.mkdir', function() {
       });
     });
   });
+
+  //not sure about error code
+  it('should not create dir with restricted symbols', function(done) {
+    var fs = util.fs();
+
+    fs.mkdir('/?', function(error) {
+      expect(error).to.exist;
+      expect(error.code).to.equal('DIRECTORY');
+      done();
+    });
+  });
+
```

Comment:
Hi, just to nitpick here,
Maybe we can try throwing promises for example:
it('(promise) should not create dir with restricted symbols', () => {
  var fsPromises = util.fs().promises;
  return fsPromises.mkdir('/?')
    .catch(error => {
      expect(error).to.exist;
      expect(error.code).to.equal('ENOENT'); 
    });
});